lang/spec.html

```diff
@@ -5372,8 +5372,8 @@ <h4>New expression</h4>
 When the class-descriptor refers to a class, the explicit-new-expr allocates
 storage for an object of the type defined by the class and initializes it by
 passing the supplied arg-list to the <code>init</code> method defined by the
-class. It is a compile error if the type-descriptor if the arg-list does not
-match the signature of the class's <code>init</code> method. If the result of
+class. It is a compile error if the arg-list does not
+match with the signature of the class's <code>init</code> method. If the result of
```

Comment:
We don't need to add "with". Can you please update accordingly?

Reply:
Removed